src/Servers/HttpSys/test/FunctionalTests/ResponseHeaderTests.cs

```diff
@@ -50,7 +50,9 @@ public async Task ResponseHeaders_ServerSendsSingleValueKnownHeaders_Success()
             }))
             {
                 // HttpClient would merge the headers no matter what
+#pragma warning disable 618
```

Comment:
Link to followup issue? They're going to break WebRequest.Create before too long.

Reply:
Was waiting for the PR to actually build

Reply:
Did you actually mean the opposite, that the source should have a link to an issue?